book/website/collaboration/academic-industry/case-studies-and-best-practices/practitioner's-hub-case-study.md

```diff
@@ -0,0 +1,24 @@
+# Case Study: Turing Way Practitioner's Hub 
+
+## Introduction to the Turing Way Practitioner's Hub 
+
+ The Turing Way Practitioners Hub is a Turing Way project that engages and involves industry experts, called Experts in Residence (EiRs), from partnering organisations. This initiative aims to advance their efforts in promoting best practices in data science and AI. The Practitioners Hub serves as a platform for cross-sector collaboration, knowledge exchange, and strategic partnerships, spanning various sectors and data science initiatives, including but not limited to the small and medium-sized (SMEs) enterprises in BridgeAI network. It is a useful example of industry-academia collaboration which could be recreated in other contexts. 
+ 
+ By leveraging a cohort-based approach, we build a shared understanding of open source principles, open data practices, cross-sector collaboration, research reproducibility, and ethical considerations in the context of data science and AI. EiRs are supported to develop case studies with stakeholders in their networks, sharing insights into the successes and challenges associated with the adoption of AI within their respective sectors. This is combined with customised training, expert consultation and collaborative opportunities, contributing to the improvement and adoption of data science practices that enhance the quality, viability, and real-world impact of data science and AI technologies in their sectors.
+
+This work is supported by Innovate UK BridgeAI. The Practitioners Hub has also received funding and support from the Ecosystem Leadership Award under the EPSRC Grant EP/X03870X/1 and The Alan Turing Institute.
+
+## Case-studies 
+
+In 2023, The Turing Way team partnered with five organisations as part of the inaurgural cohort of this programme: 
+- British Antarctic Survey
+- Genomics England
+- Office of National Statistics
+- Energy Systems Catapult
+- Digihaul
+
+During the course of the 2023 programme the Turing Way team co-created case studies about best practice open source science and AI adoption with each of the EiRs, with the help of scientific writer Stuart Gillespie. A meta case study was also produced to bring together learnings about organisational adoption of open and and responsible AI from each participating organisation ((https://zenodo.org/records/10777730)). Explore these case studies and other materials from the first cohort here: zenodo.org/communities/the-turing-way-practitioners.
+
+The case studies were created using the following framework (* insert link to TW page here). 
```

Comment:
We can probably leave this out for now and mark it as work to do once the framework is ready for publication, so that we can have a pull request ready for review asap?

Reply:
Yes sounds good